src/core/Akka.Cluster/ClusterDaemon.cs

```diff
@@ -2172,6 +2175,10 @@ private void ShutdownSelfWhenDown()
                     SendGossipRandom(MaxGossipsBeforeShuttingDownMyself);
                     Shutdown();
                 }
+                else
+                {
+                    _selfDownCounter++;
```

Comment:
Makes sense - gives the node a chance to gossip a few times prior to termination.